src/main/webapp/game.js

```diff
@@ -10,11 +13,52 @@ const CSS_MESSAGE_CLASS_DICT = {
 };
 // TODO: @salilnadkarni, replace with userid from cookie (in datastore)
 const USER_ID = "_" + Math.random().toString(36).substr(2, 9);
+var videoId = "";
```

Comment:
Change from var to let according, see [google's javascript style guide section 5.1.1](https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations)

Reply:
it's not a local variable, it is global @salnad